extension/scripts/features/shopsfillmax/ttShopsFillMax.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+(async () => {
+	const feature = featureManager.registerFeature(
+		"Fill Max",
+		"shops",
+		() => settings.pages.shops.fillMax,
+		null,
+		addFillMax,
+		removeFillMax,
+		{
+			storage: ["settings.pages.shops.fillMax"],
+		},
+		null
+	);
+
+	async function initialiseListeners() {
+		if (!(await checkMobile())) return;
+		addFetchListener((event) => {
+			if (!feature.enabled) return;
+
+			const { page, json, fetch } = event.detail;
+			if (page !== "bazaar") return;
+			if (new URL(fetch.url).searchParams.get("step") !== "getBazaarItems") return;
+
+			maxBuyListener({}, true);
```

Comment:
You are confusing me with these parameters.

- The first one is defaulted to a string, why pass an object (or have a string as default).
- There isn't even a second one.